Bylaws.md

```diff
@@ -157,11 +165,11 @@ The public may follow the Forum’s activities by reading all postings on the Pu
 
- 3. Manner of holding officer elections: If a single individual is nominated for a position, the Forum will hold a ballot prepared and submitted by the Chair, with the exception of not requiring endorsers, to confirm appointment of the nominee. For the confirmation ballot, each Voting Member is entitled to a single vote regardless of the number of participating Voting Member representatives or whether the Voting Member is categorized as a Certificate (or Root Certificate) Issuer or a Certificate Consumer. Only designated Voting Representatives are allowed to cast votes on behalf of Voting Members. If multiple votes are received from a Voting Member’s representatives, the last vote submitted during the voting period is considered the Voting Member’s vote. The single nominee is considered confirmed if a majority of the Voting Members who vote are in favor of the appointment, regardless of the number of votes cast and irrespective of whether 2/3 of the Certificate (or Root Certificate) Issuers or 1/2 of the Certificate Consumers approve appointment of the nominee.
+ 3. Manner of holding officer elections: If a single individual is nominated for a position, the Forum will hold a ballot prepared and submitted by the Chair, with the exception of not requiring endorsers, to confirm appointment of the nominee. For the confirmation ballot, each Voting Member is entitled to a single vote regardless of the number of participating Voting Member representatives or whether the Voting Member is categorized as a Certificate Issuer or a Certificate Consumer. Only designated Voting Representatives are allowed to cast votes on behalf of Voting Members. If multiple votes are received from a Voting Member’s representatives, the last vote submitted during the voting period is considered the Voting Member’s vote. The single nominee is considered confirmed if a majority of the Voting Members who vote are in favor of the appointment, regardless of the number of votes cast and irrespective of whether 2/3 of the Certificate Issuers or 1/2 of the Certificate Consumers approve appointment of the nominee.
```

Comment:
Should we not use the same voting conditions here? 
So, `50%+1` like in 2.3.6 instead of `1/2`

```suggestion
 3. Manner of holding officer elections: If a single individual is nominated for a position, the Forum will hold a ballot prepared and submitted by the Chair, with the exception of not requiring endorsers, to confirm appointment of the nominee. For the confirmation ballot, each Voting Member is entitled to a single vote regardless of the number of participating Voting Member representatives or whether the Voting Member is categorized as a Certificate Issuer or a Certificate Consumer. Only designated Voting Representatives are allowed to cast votes on behalf of Voting Members. If multiple votes are received from a Voting Member’s representatives, the last vote submitted during the voting period is considered the Voting Member’s vote. The single nominee is considered confirmed if a majority of the Voting Members who vote are in favor of the appointment, regardless of the number of votes cast and irrespective of whether 2/3 of the Certificate Issuers or fifty percent (50%) plus one (1) of the Certificate Consumers approve appointment of the nominee.
```

Reply:
I agree with the suggestion. I will leave it open for any other possible comments, especially from @timfromdigicert who will work on the elections language.